style.css

```diff
@@ -10,4 +10,11 @@ td {
   width: 50px;
   height: 50px;
   border: 1px solid black;
+}
+
+.alive {
+  background-color: orange;
+}
+.dead {
+  background-color: #fff;
 }
```

Comment:
This seems unnecessary, it is white by default. Also even if it is white, the other class has named color, so why this one has hex color. Consistency for the win.